sites/partners/src/components/applications/PaperApplicationDetails/sections/DetailsHouseholdMembers.tsx

```diff
@@ -14,11 +14,11 @@ const DetailsHouseholdMembers = ({ setMembersDrawer }: DetailsHouseholdMembersPr
   const application = useContext(ApplicationContext)
 
   const householdMembersHeaders = {
-    name: t("t.name"),
-    relationship: t("t.relationship"),
-    birth: t("application.household.member.dateOfBirth"),
-    sameResidence: t("application.add.sameResidence"),
-    workInRegion: t("application.details.workInRegion"),
+    name: "t.name",
```

Comment:
This was throwing console errors - it's a little odd but the function these are passed into just takes the string which it passes into `t()`